include/leveled.hrl

```diff
@@ -80,9 +89,9 @@
                         timestamp :: integer()}).                      
 
 -record(manifest_entry,
-                        {start_key :: tuple() | undefined,
-                        end_key :: tuple() | undefined,
-                        owner :: pid()|list(),
+                        {start_key :: tuple(),
+                        end_key :: tuple(),
+                        owner :: pid(),
```

Comment:
There used to be some unit tests that generated artificial manifest entries, used the owner to identify manifest_entry when asserting. To make this identification easier, the owner was given as a string, rather than spawning PIDs to create artificial manifest entries. 

Going forward, on principle the types should be accurate, even if that takes a lit more effort to generate test cases.